pkg/crypto/edg.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package crypto
```

Comment:
struct names `Secp256k1EDG` and `Secp256r1EDG` look identical...

are names `EDGSecp256K` and `EDGSecp256R` bit better? how about `EDGSecp256_K` and `EDGSecp256_R`?

Reply:
fixed